docs/components/HomePageCard.js

```diff
@@ -0,0 +1,59 @@
+// @flow strict
+import { Badge, Box, Card, Flex, Heading, TapArea, Text } from 'gestalt';
+import { type Node } from 'react';
+
+type Props = {|
+  image: Node,
+  description: string,
+  title: string,
+  color: string,
+  href: string,
+  isNew?: boolean,
+|};
+
+function HomePageCard({ image, description, isNew, title, color, href }: Props): Node {
+  return (
+    <TapArea href={href} role="link" accessibilityLabel={`${title} page`}>
+      <Box minWidth={280}>
+        <Card>
+          <Flex direction="column" height={320}>
+            <Box
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
+              height="60%"
+              width="100%"
+              dangerouslySetInlineStyle={{
+                __style: {
+                  backgroundColor: `var(--color-${color}-450)`,
```

Comment:
@ayeshakmaz interesting - should we not allow these colors to be set as `color` on `<Box />`?

Reply:
I'm curious about this as well

Reply:
For now, we're limiting Box colors to semantic tokens only to avoid the situation we were in before with loads of color options being available